### ISSUE TITLE
inform users waiting for their test result about teleTAN 

### ISSUE THREAD

**Issue by achisto** [corona-warn-app/cwa-wishlist#287]
## Current Implementation
When a user adds a test by scanning its QR-Code right after the testing procedure but that test's result is never transmitted to CWA they will not know that
- sometimes test results are not transmitted due to labs not connected to the CWA-infrastructure
- if a test result is not transmitted they are still able to warn their contacts

## Suggested Enhancement
There should be a notice on the "waiting for result" screen which reminds the user
- to contact local health authorities in case a result is not shown within e.g. 4 days
- that after receiving a positive result by phone/sms but **not** in CWA there is the option to request a teleTAN and warn contacts

Optional, that notice could only be shown after e.g. 3 days have passed since the test was added. In that case there should be a push-notification.

## Expected Benefits
I personally know two people, who received a positive result by SMS while the CWA didn't show the positive result for several days. Both did not know they could call the TAN-hotline and warn their contacts by requesting a teleTAN. 

There is no way of finding out about this alternative because not only does CWA fail to show a hint on the test-screen but, even worse, after having added the test there is no way of going back to the "add your result"-screen in order to find out about other verification methods. 

While there is still a significant number of labs not integrated into the CWA-infrastructure I am afraid that there could be a lot of people willing to share their result who simply don't know how to accomplish that. I don't have a lot of contacts and still know about two of those people. Adding a hint to the "waiting for result"-screen could significantly improve the number of people warning other users.

**Comment by Ein-Tim** [comment]
Please see also: https://github.com/corona-warn-app/cwa-wishlist/issues/270

**Comment by heinezen** [comment]
Hello @achisto ,

Thank you for your suggestions. We have added your feedback to s ticket in our internal Jira (EXPOSUREAPP-3953) where it will be investigated by the developers and forwarded to be discussed by the RKI.

Since your feature request ist very similar to https://github.com/corona-warn-app/cwa-wishlist/issues/270 and https://github.com/corona-warn-app/cwa-wishlist/issues/282, we have decided to group the three issues together in one ticket. I will also close your issue as a duplicate of https://github.com/corona-warn-app/cwa-wishlist/issues/270 to keep the discussion in a central place. However, your initial post will not be lost as we will link it in the mentioned issue. You are also free to copy your post over as a comment if you want.

We will update you on news from the development team when they become available.

Regards,
CH

----
Corona-Warn-App Open Source Team
